### PR TITLE
Support copy-assets Nx target on Windows

### PR DESCRIPTION
**Pull request by KK4ito**
Hi there, 

I have a possible solution for the issue #2383. The plugin run-script-os allows us to write npm platform specific scripts.

I adjusted every npm script which contains the "nx run client:copy-assets" to use the platform specific command.
I also had to duplicate the nx command "copy-assets".

Thanks for a feedback.

### CODE REVIEW

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thanks a lot for providing a solution @KK4ito.

Isn't there a more elegant solution? 🤔 I don't like so much that the paths are in there twice now. This way, unfortunately, it is often forgotten that it must be adjusted in both locations.

**Comment by KK4ito** [comment]
Thanks for the feedback @dtslvr.
I changed the solution by using the "shx" plugin.

File: package.json [hunk 1]
```diff
@@ -194,6 +194,7 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "replace-in-file": "7.0.1",
+    "shx": "^0.3.4",
```
Comment (dtslvr) [inline package.json:197]: That looks promising. Can you fix the version (remove `^`) and run `yarn install` again?

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KK4ito) [reply package.json]: done, see commit c0d89362625faf4dd0e00e0fb94df6d57fd4aa3e

**Review by dtslvr — APPROVED** [review]
Thank you Kevin! 👍🏻

**Pull request merged**